src/layouts/articleLayout.tsx

```diff
@@ -8,6 +8,7 @@ import SEO from '../components/seo'
 import { graphql } from 'gatsby'
 import MDXRenderer from 'gatsby-plugin-mdx/mdx-renderer'
 import { useNavigate } from '@reach/router'
+import { MDXProvider } from "@mdx-js/react"
```

Comment:
@rahul3v This is not the right place to define new MDXProvider components. It should be defined in `src/components/customMdx/index.tsx`
Also, this is a path issue which needs to be resolved on the plugin level, which you can find inside `plugins/` folder

Reply:
ok @nilubava, Well there are too many problems, even if we re fix the only file root path, the link still breaks , so i have to write code to run specifically for local run, see the next commit